custom/inddex/templates/inddex/partials/report_table.html

```diff
@@ -1,7 +1,7 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-<table id="report_table_{{ report_table.slug }}" class="table table-striped datatable" {% if pagination.filter %} data-filter="true"{% endif %}>
+<table id="report_table_{{ report_table.slug }}" class="table table-striped datatable" {% if pagination.filter %} data-filter="true"{% endif %} data-report-table="{{ report_table|JSON }}">
```

Comment:
Most places where we use the `|JSON` filter in the context of an attribute value, the attribute value is enclosed in single quotes (`'`) rather than doulbe (`"`). Not sure if that's important or not. JSON data often contains double quotes.

Aside: it's possible for it to contain single quotes as well, so maybe there are situations where more escaping is needed? I don't know if `|JSON` does that.

Reply:
Good point, and I don't see quote escaping in the template tag definition. I'll do some testing.

Reply:
Did some testing locally, and it properly handles both kinds of quotes in the data (both in keys and values), whether the `{{ data|JSON }}` is enclosed in single or double quotes.